app/models/trip.rb

```diff
@@ -1,21 +1,37 @@
 class Trip < ApplicationRecord
   belongs_to :school
+  has_many :trip_families
+  has_many :families, through: :trip_families
 
   validate :start_date_in_the_past
   validate :start_date_after_end_date
-  validates :total_girls,
+  validates(
+    :end_date,
+    :start_date,
     :total_boys,
-    :total_teachers,
     :total_bus_drivers,
-    :start_date,
-    :end_date,
-    presence: true
+    :total_girls,
+    :total_teachers,
+    presence: true,
+  )
 
   delegate :name, to: :school, prefix: true
 
-  scope :current, -> { where("end_date > ?", Time.current) }
-  scope :past, -> { where("end_date <= ?", Time.current) }
-  scope :by_start_date, -> { order("start_date") }
+  def self.current
+    where("end_date > ?", Time.current)
+  end
+
+  def self.past
+    where("end_date <= ?", Time.current)
+  end
+
+  def self.by_start_date
+    order("start_date")
```

Comment:
A suggestion to change to using a symbol: `order(:start_date)`?

Reply:
Done in sha `d57af14`